src/python/espressomd/electrostatics.pyx

```diff
@@ -38,7 +38,15 @@ IF ELECTROSTATICS == 1:
         def _set_params_in_es_core(self):
             raise Exception(
                 "Subclasses of ElectrostaticInteraction must define the _set_params_in_es_core() method.")
-    
+
+        def _check_for_electroneutrality(self):
```

Comment:
this function is duplicate to the one in electrostatic_extensions

Reply:
Autopep8 will be applied anyway with the release of 4.0 right?